use_case_examples/hybrid_model/infer_hybrid_llm_generate.py

```diff
@@ -39,7 +39,7 @@
         module_names,
         server_remote_address="http://0.0.0.0:8000",
         model_name=f"{model_name}",
-        verbose=False,
+        verbose=True,
```

Comment:
I wouldn't put it as default as it breaks the streaming decoding.

Reply:
ah I didn't realize that. I will revert.

Reply:
(I mean it works but it's just lost in the flow of stdout)